### PR TITLE
refactor:remove `common_error::prelude`

### PR DESCRIPTION
**Pull request by Gump9**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Remove `prelude` mod from `common-error` crate. Most re-exports in the `prelude` mod is seldom been used.
Remove the `prelude` mod and replace usages of prelude by using items explicitly.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: src/common/error/src/lib.rs [hunk 1]
```diff
@@ -17,16 +17,19 @@ pub mod format;
...
-    pub use snafu::{Backtrace, ErrorCompat};
+pub const INNER_ERROR_CODE: &str = "INNER_ERROR_CODE";
+pub const INNER_ERROR_MSG: &str = "INNER_ERROR_MSG";
 
-    pub use crate::ext::{BoxedError, ErrorExt};
-    pub use crate::format::DebugFormat;
-    pub use crate::status_code::StatusCode;
+// pub mod prelude {
+//     pub use snafu::prelude::*;
+//     pub use snafu::{Backtrace, ErrorCompat};
```
Comment (evenyag) [inline src/common/error/src/lib.rs:25]: suggestion: remove the prelude mod
```suggestion
```

File: src/common/error/src/lib.rs [hunk 2]
```diff
@@ -17,16 +17,19 @@ pub mod format;
...
-    pub const INNER_ERROR_CODE: &str = "INNER_ERROR_CODE";
-    pub const INNER_ERROR_MSG: &str = "INNER_ERROR_MSG";
-}
+//     pub use crate::ext::{BoxedError, ErrorExt};
+//     pub use crate::format::DebugFormat;
+//     pub use crate::status_code::StatusCode;
+
+//     pub const INNER_ERROR_CODE: &str = "INNER_ERROR_CODE";
+//     pub const INNER_ERROR_MSG: &str = "INNER_ERROR_MSG";
+// }
```
Comment (evenyag) [inline src/common/error/src/lib.rs:33]:
```suggestion
```

Comment (Gump9) [reply src/common/error/src/lib.rs:33]: ok i will remove them

**Review by evenyag — APPROVED** [review]
LGTM

**Comment by evenyag** [comment]
Oops, sorry, could you please resolve the conflicts? @Gump9 😢

**Comment by Gump9** [comment]
> Oops, sorry, could you please resolve the conflicts? @Gump9 😢

👌,i will

**Review by waynexia — APPROVED** [review]

**Pull request merged**